pkg/kubelet/kubeletconfig/checkpoint/store/fsstore_test.go

```diff
@@ -71,6 +60,7 @@ func TestFsStoreInitialize(t *testing.T) {
 	if err != nil {
 		t.Fatalf("fsStore.Initialize() failed with error: %v", err)
 	}
+	defer func() { _ = store.fs.RemoveAll(store.dir) }()
```

Comment:
Rather than re-defining this anonymous function over and over, can we define it once and pass in `store` as a parameter?

Reply:
Thanks @ehashman  ! 
done.
please review again.   : )